src/main/webapp/js/AdminCohortListView.js

```diff
@@ -0,0 +1,54 @@
+class AdminCohortListView extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            cohortName: '',
+            cohorts: []
+        };
+    }
+
+    render() {
+        // Is it possible for no cohorts to be returned?
+        return (
+            <div>
+                <h2> Cohorts </h2>
+                <textarea value={this.state.cohortName} onChange={this.userChange.bind(this)}></textarea>
+                <button> Submit </button>
+                <div>
+                    {this.state.cohorts.length == 0 && (<p> Sorry, we could not find a cohort with that name. </p>)}
+                    {this.state.cohorts.length > 0 /* would we do some sort of for-each here? */
+                        && ( <ol> {cohortName} </ol> )
+                    }
+                </div>
+            </div>
+        )
+    }
+
+    componentDidMount() {
+        this.getCohorts();
+    }
+
+    userChange(event) {
+        this.setState({cohortName: event.target.value});
+    }
+
+    getCohorts() {
+        const url = `/cohort?cohortId=${this.props.match.params.cohortId}`;
```

Comment:
For this we can just fetch the cohorts without any filtering(because it should display a list of all of the cohorts.
```suggestion
        const url = '/cohort-list';
```